app/server/lib/DocApi.ts

```diff
@@ -1536,6 +1534,22 @@ export class DocWorkerApi {
       return res.status(200).json(docId);
     }));
 
+    this._app.post('/api/docs/:docId/copy', canView, expressWrap(async (req, res) => {
+      const userId = getUserId(req);
+
+      const parameters: {[key: string]: any} = req.body;
```

Comment:
Is the type annotation required here? It seems so generic as to be almost useless.